llvm/include/llvm/Frontend/OpenMP/OMPContext.h

```diff
@@ -174,6 +177,8 @@ struct OMPContext {
 
   BitVector ActiveTraits = BitVector(unsigned(TraitProperty::Last) + 1);
   SmallVector<TraitProperty, 8> ConstructTraits;
+  static int DeviceNum;
+  static StringRef DeviceNumID;
```

Comment:
Not sure static is correct here

Reply:
I used static to ensure that all instances of OMPContext share the same value. Any other suggestions?

Reply:
This is the main question. Can it be affected somehow per file/function, or is it always the same for the whole compilation unit?